Neos.FluidAdaptor/Classes/View/TemplatePaths.php

```diff
@@ -356,20 +358,6 @@ protected function sanitizePath($path)
         return $path;
     }
 
-    /**
-     * @param string $packageKey
-     * @return string|null
-     */
-    protected function getPackagePrivateResourcesPath($packageKey)
-    {
-        if (!$this->packageManager->isPackageAvailable($packageKey)) {
-            return null;
-        }
-        $packageResourcesPath = $this->packageManager->getPackage($packageKey)->getResourcesPath();
-
-        return Files::concatenatePaths([$packageResourcesPath, 'Private']);
-    }
-
```

Comment:
Why is this removed specifically? Looks unused but so do other methods in there.

Reply:
yes its unused and phpstan doesnt like it. Other unused methods might be public which is okay.